curriculum/dictionaries/portuguese/comments.json

```diff
@@ -1,110 +1,110 @@
 {
-  "hyek8f": "24.44 in Celsius",
```

Comment:
@nhcarrigan any idea how these changes made it into this PR?

Reply:
It seems correct, in crowdin the comments are in the curriculum project
![image](https://user-images.githubusercontent.com/26656284/126138159-ce7e99b2-af31-4e48-942c-3afa2932be7b.png)

Reply:
Oh, wow, I should have realised that.  Thanks @ieahleen.